### ISSUE TITLE
Show status of 2FA usage in user management

### ISSUE THREAD

**Issue by j-ed** [nextcloud/server#13395]
## Expected behavior
The 2FS support has greatly been improved with NC 15.x, but it is currently not possible for an administrator to  check if a user has enabled any kind of 2FA for his account. Best would be if this information could be seen at a central point, like e.g. user management.

Such an indicator could e.g. look like this:

![20190106_nc_user_management-2](https://user-images.githubusercontent.com/2572764/50740062-615c4580-11e9-11e9-9626-dbd137991658.png)

The color of the icon could e.g. indicate the status of a 2FA activation, **red**- not enabled/activated, **yellow** - activated but not yet used, **green** - everything is ok.

By clicking on the icon the enabled 2FA apps could be shown, like **twofactor_gateway: telegram**, **Two factor U2F** or **twofactor_nextcloud_notification** etc.

#### Server Configuration
OS: Linux 3.16.50
Web server: Apache2 2.4.37
Database: MariaDB 10.2.19
PHP version: 7.2.13
Nextcloud version: 15.0.0

#### Client Configuration
Browser: Mozilla Firefox 64.0
Operating system: Windows 10

**Comment by MorrisJobke** [comment]
cc @ChristophWurst

**Comment by ChristophWurst** [comment]
Should be doable with the stateful providers data.

**Comment by TecJon** [comment]
Good idea @j-ed. However, I find the proposed colors rather flashy. How about a gray icon if it is used (maybe add a small check to the icon) and no icon if not enabled/not used?

**Comment by j-ed** [comment]
@TecJon The last word has always the NC design team ;-) Unfortunately it is not possible to show 3 different states with an on/off logic. But let's see what other ideas are coming up on this.

**Comment by szaimen** [comment]
I suppose this is still valid?

**Comment by j-ed** [comment]
Yes, that issue is still valid.
